### ISSUE TITLE
Unable to use @dolbyio/comms-sdk-react-native, @millicast/sdk and react-native-webrtc together

### ISSUE THREAD

**Issue by DannyBiezen** [DolbyIO/comms-sdk-react-native#61]
I am using the following packages in an Expo managed project:
* `"@dolbyio/comms-sdk-react-native": "^3.7.0-beta.1",`
* `"@millicast/sdk": "^0.1.35",`
* `"react-native-webrtc": "^106.0.0-beta.7",`,

When trying to create an expo - EAS build I get a lot of `Duplicate class`  errors:
![image](https://user-images.githubusercontent.com/94181463/205291472-2496681e-eb88-4a32-aeaa-b4b06ae68ec4.png)

I need to use both SDK's in my app, is this supported? If yes, what tweaks to the native code do I need to make to get this working?

**Comment by FabienLavocat** [comment]
Hi @DannyBiezen, today, both SDKs (Communications and Real-Time Streaming) are using the same dependency (webrtc) with different versions, so it creates a conflict. You will not be able to load the two SDKs in the same application unfortunately.

On the Millicast side, are you trying to publish or subscribe to a stream?

**Comment by DannyBiezen** [comment]
Thanks for your reply!

Currently we only need to use Millicast to subscribe to a stream. Having the ability to watch a real time stream and being able to join a video call is crucial to our app. Do you know if this issue will be worked on in the near-future?

**Comment by FabienLavocat** [comment]
Yes, this is something we are working on at the moment, I just don't have an ETA, but I'd be happy to discuss with you regarding your project and see if there is anything we can do in the meantime. Don't hesitate to reach out and we can discuss this on a call: fabien.lavocat @ dolby . com

**Comment by DannyBiezen** [comment]
That's great to hear! I'll only need to integrate with Dolby in a couple of weeks/months so will definitely reach out when I start to integrate the Dolby SDK. Thanks

**Comment by FabienLavocat** [comment]
Sounds good, if you want to discuss your project before starting the integration, send an email my way.

**Comment by Drengel1990** [comment]
I have the same problem!!!

**Comment by wt-mcare** [comment]
Same problem for me on Expo with @dolbyio/comms-sdk-react-native 3.7.0 and react-native-webrtc 106.0.4
Dolbyio solution is working great but can't build with webrtc lib installed.
